tests/db/model/bot/utils.js

```diff
@@ -10,16 +10,20 @@
  * tests/db/model/bot/utils.js
  */
 'use strict';
-
+const path = require('path');
+const fs = require('fs');
 const tu = require('../../../testUtils');
 
 const testStartTime = new Date();
 const n = `${tu.namePrefix}TestBot`;
 const n2 = n+'NonActive';
+const mt = path.join(__dirname, './uiBlob');
+const uiBlob = fs.readFileSync(mt);
 
 const standard = {
   name: n,
   url: 'http://www.bar.com',
+  ui: uiBlob,
```

Comment:
An update db test for this field would be great, but other than that, I am fine merging this PR.